home-work-app/src/app/pages/home/components/AddMovieLink/StyledLink.styled.tsx

```diff
@@ -1,11 +1,10 @@
-import styled, { StyledComponent } from 'styled-components';
-import { stylesConfig } from '../../../../configs/style.configs';
+import styled, { StyledComponent, ThemeProps, DefaultTheme } from 'styled-components';
 
 export const StyledLink: StyledComponent<'div', {}> = styled.div.attrs({
```

Comment:
`div` with `role: button` looks realy awkward. I should be a `button`.
Also, I think this component should be called `AddMovieButton`.
Rework it with `className` prop and move its content inside.
And then in `AddMovieLink` return value be smth like that:
```
<>
   <AddMovieButton onClick={handleAddButtonClick} />
   {isModalOpen && (
      <ModalPortal modalRoot={modalRoot}>
           <AddMovieModal onClose={clickHandler}/>
      </ModalPortal>
   )}
</>
```